x/emissions/module/rewards/rewards_test.go

```diff
@@ -1,6 +1,7 @@
 package rewards_test
 
 import (
+	stdlog "log"
```

Comment:
can we use `s.T().Log()` to use the built in logging package in the testing package rather than importing something new?